web/src/hotkey.js

```diff
@@ -447,6 +447,11 @@ export function process_enter_key(e) {
         return true;
     }
 
+    if (e.target.id === "send_later") {
+        $("#send_later i").trigger("click");
+        return true;
+    }
```

Comment:
Could this be replaced with just setting `role="button"` on the send later button, to have the next block cover that case?

Reply:
No, since the popover is triggered by clicking on `i` and not the button. Added a comment to explain it as well.

Reply:
OK, cool, I could imagine us giving that thing more of a class name but it seems OK for now.